Datagrid/ProxyQuery.php

```diff
@@ -74,6 +74,24 @@ public function execute(array $params = array(), $hydrationMode = null)
             $queryBuilder->resetDQLPart('orderBy');
         }
 
+        /* By default, always add a sort on the identifier fields of the first
+            * used entity in the query, because RDBMS do not guarantee a
```

Comment:
I changed the comment to something more reasonable. As per the `//`, is is best suited for when your comment is composed of things that are independent and could thus be moved around, but here, I have a long sentence, so…